src/FakerDotNet/Faker.cs

```diff
@@ -22,6 +22,7 @@ public static class Faker
         public static ICommerceFaker Commerce { get; } = Container.Commerce;
         public static ICompanyFaker Company { get; } = Container.Company;
         public static ICompassFaker Compass { get; } = Container.Compass;
+        public static ICreditCardFaker creditCard { get; } = Container.CreditCard;
```

Comment:
These properties should all be **PascalCase**

```suggestion
        public static ICreditCardFaker CreditCard { get; } = Container.CreditCard;
```